analyzer/src/main/kotlin/managers/SBT.kt

```diff
@@ -69,10 +69,17 @@ class SBT : PackageManager() {
              // Note that "sbt sbtVersion" behaves differently when executed inside or outside an SBT project, see
              // https://stackoverflow.com/a/20337575/1127485.
              val workingDir = definitionFiles.first().parentFile
+
+             // See https://github.com/sbt/sbt/issues/2695.
+             var sbtLogNoformat = "\"-Dsbt.log.noformat=true\""
+             if (!OS.isWindows) {
+                 sbtLogNoformat = sbtLogNoformat.removeSurrounding("\"")
+             }
```

Comment:
I was thinking how to not make this a var and came up with this:
```
        val test = buildString {
            val surrounding = { if (OS.isWindows) "\"" else "" }
            append(surrounding)
            append("string")
            append(surrounding)
        }
```

But maybe that's overengineering and we should simple make it like this? Might be the easiest to read and understand.
```
val sbtLogNoformat = if(OS.isWindows) "\"...\"" else "..."
```

Reply:
Or:

             val sbtLogNoformat = "-Dsbt.log.noformat=true".let {
                 if (OS.isWindows) {
                     "\"$it\""
                 } else {
                     it
                 }
             }

?

Reply:
Now it would be nice to have a 'letIf(condition)' ;)
But yes, that would be better.